lib/cloudware/models/machine.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'models/application'
+require 'models/concerns/provider_client'
+require 'providers/AWS'
```

Comment:
This shouldn't be required here AFAICS - looks like it's always requiring the AWS provider (which affects start up time, obvs.)